# This file is part of HiGFXback

# requires
REQUIRES="a52dec-build autotools-wrappers-lt-build"

pkg-config --exists --print-errors $REQUIRES || exit 1

# configure (ac-2.69; am-1.11; lt-2.4.2)
libtoolize -c -f; aclocal -I m4; autoconf; autoheader -f; automake -a -c
./configure --prefix=

# build
make

# install
install -d $DESTDIR/bin
install .libs/mpeg3cat $DESTDIR/bin
install .libs/mpeg3dump $DESTDIR/bin
install .libs/mpeg3peek $DESTDIR/bin
install .libs/mpeg3toc $DESTDIR/bin
install -d $DESTDIR/include/mpeg3
install -m 644 libmpeg3.h $DESTDIR/include/mpeg3
install -m 644 mpeg3private.h $DESTDIR/include/mpeg3
install -d $DESTDIR/lib
install .libs/libmpeg3.so.1.* $DESTDIR/lib/libmpeg3.so.1
ln -sf libmpeg3.so.1 $DESTDIR/lib/libmpeg3.so
install -d $DESTDIR/lib/pkgconfig
install -m 644 libmpeg3.pc $DESTDIR/lib/pkgconfig

# build.pc
install -d $DESTDIR/share/pkgconfig
cat > $DESTDIR/share/pkgconfig/libmpeg3-build.pc << EOF
Name: libmpeg3
Version: 20140922
Description: MPEG streams decoding library
Requires: $REQUIRES

devel=\\
/include/mpeg3/libmpeg3.h \\
/include/mpeg3/mpeg3private.h \\
/lib/libmpeg3.so \\
/lib/pkgconfig/libmpeg3.pc

exec=\\
/bin/mpeg3cat \\
/bin/mpeg3dump \\
/bin/mpeg3peek \\
/bin/mpeg3toc \\
/lib/libmpeg3.so.1
EOF

exit
--- libmpeg3-20140922.orig/configure.ac
+++ libmpeg3-20140922/configure.ac
@@ -1,5 +1,6 @@
 # Process this file with autoconf to produce a configure script.
 AC_INIT(libmpeg3, 1.8)
+AC_CONFIG_MACRO_DIR([m4])
 AM_INIT_AUTOMAKE([dist-bzip2])
 AC_PREREQ(2.53)
 AM_CONFIG_HEADER([config.h])
